pytests/tests/test_check_skip.py

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright (C) 2019 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the GNU General Public License v2 (the "License");
+# you may not use this file except in compliance with the License. The terms
+# of the License are located in the COPYING file of this distribution.
+#
+#   Author: Shreenidhi Shedi <sshedi@vmware.com>
+
+import os
+import tempfile
+import pytest
+import shutil
+import subprocess
+
+isNotPhoton = -1
+try:
+    isNotPhoton = subprocess.check_call("uname -a | grep -i photon > /dev/null", shell = True)
+except Exception as e:
+    isNotPhoton = 1
+
+def run_cmd(utils, cmd, retval):
+    ret = utils.run(cmd)
+    if retval == 0:
+        assert(ret['retval'] == 0)
+    else:
+        assert(ret['retval'] != 0)
+
+@pytest.fixture(scope='module', autouse=True)
+def setup_test(utils):
+    yield
+    teardown_test(utils)
+
+def teardown_test(utils):
+    pass
+
+def test_check_skipconflicts(utils):
+    cmd = [ 'tdnf', '--config', '/etc/tdnf/tdnf.conf', 'check', '--skipconflicts']
```

Comment:
@sshedi, why are you referring to system config instead of test-repo config here?